armi/tests/test_lwrInputs.py

```diff
@@ -63,7 +63,6 @@ def test_loadC5G7(self):
             streamVal = mock.getStdout()
             self.assertIn("UraniumOxide", streamVal, msg=streamVal)
             self.assertIn("SaturatedWater", streamVal, msg=streamVal)
-            self.assertIn("invalid settings: fakeBad", streamVal, msg=streamVal)
```

Comment:
Ah I see, I didn't even think that it was there on purpose. This sort of behavior (checking that invalid settings are caught in settings validation) feels like it should be covered elsewhere, yeah? 

Just want to make sure we aren't losing something by ripping this out.

Reply:
We have this in several other places, not to worry.